.gitmodules

```diff
@@ -4,7 +4,7 @@
 [submodule "lib/eigenlayer-contracts"]
 	path = lib/eigenlayer-contracts
 	url = https://github.com/Layr-labs/eigenlayer-contracts
-	branch = dev
+	branch = release-dev/multichain
```

Comment:
We can change this to `dev` once `release-dev/multichain` has been merged in the core repo